docs/glossary.rst

```diff
@@ -9,6 +9,15 @@ Glossary
 .. glossary::
    :sorted:
 
+   alias
+   aliases
+      An abbreviated version of a commonly used function that is
+      intended primarily for interactive use. Aliases are named with a
+      trailing underscore. For example, `~plasmapy.formulary.speeds.va_`
+      is an alias for `~plasmapy.formulary.speeds.Alfven_speed`). For
+      further details, please refer to the :ref:`contributor guide's
+      section on aliases <aliases>`.
```

Comment:
```suggestion
      An abbreviated version of a commonly used function.  For
      example, `~plasmapy.formulary.speeds.va_` is an alias for
      `~plasmapy.formulary.speeds.Alfven_speed`.  Aliases are
      named with a trailing underscore. 
      
      For further details, please refer to the :ref:`contributor guide's
      section on aliases <aliases>`.
```

Just trying to match this up with the rewording we did to the Aliases section.

Reply:
Since it's a glossary entry and fairly short, I'm inclined to keep it a single paragraph.

Reply:
I'm fine with that. I just wanted to get that confusing "interactive use" phrase out of there and replace it with the same language that we put into the aliases section.